sim/warrior/fury/fury.go

```diff
@@ -91,6 +91,11 @@ func (war *FuryWarrior) GetMasteryBonusMultiplier() float64 {
 	return 1 + (11.2+5.6*war.GetMasteryPoints())/100
 }
 
+func (war *FuryWarrior) GetMasteryBonusMultiplierFromMasteryRating(masteryRating float64) float64 {
+	masteryPoints := core.MasteryRatingToMasteryPoints(masteryRating)
+	return 1 + (11.2+5.6*masteryPoints)/100
```

Comment:
Can we make sure this line and line 91 are not basically identical? Can we make it reusable?